tests/conftest.py

```diff
@@ -43,8 +43,8 @@ def create_service(client):
     with client.application.app_context():
         # Setup code to create the object in the database
         user, created = get_or_create(db.session, User, username="mehmet") # Adjust fields as necessary
-        get_or_create(db.session, Service, name="Github Demo", type="Github", desc="description", user_id=user.id)
-
+        service, created = get_or_create(db.session, Service, name="Github Demo", type="Github", desc="description", user_id=user.id)
+        service.token = "43b901cba25c458a9175ef119a9603bb"
```

Comment:
Is it your actual GitHub token or the one generated by our service?

Reply:
It was generated by our service and then I got it from there.